Emby.Dlna/ContentDirectory/ControlHandler.cs

```diff
@@ -1443,7 +1051,7 @@ private QueryResult<ServerItem> GetNextUp(BaseItem parent, InternalItemsQuery qu
                 {
                     Limit = query.Limit,
                     StartIndex = query.StartIndex,
-                    UserId = query.User.Id
+                    UserId = query.User!.Id
```

Comment:
Handle this safely?

Reply:
Safely? It can't be null unless someone changes the whole call stack. Seems very redundant to handle it safely.

Reply:
In that case add a comment explaining that
`!` should rarely be used and every time we use it we should explain why it's safe to do so (unless you do it in the dispose method)